tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -54,9 +54,9 @@ Status ModularFileSystem::NewRandomAccessFile(
   return StatusFromTF_Status(plugin_status.get());
 }
 
-Status ModularFileSystem::NewWritableFile(
-    const std::string& fname,
-    std::unique_ptr<WritableFile>* result /*, TransactionToken* token */) {
+Status ModularFileSystem::NewWritableFile(const std::string& fname,
+                                          std::unique_ptr<WritableFile>* result,
+                                          TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `result`.

Reply:
I can do that but it means that everywhere these methods used needs to be modified since we can't have default arguments in the middle of the arg list. Do you want me to make this change?

Reply:
Oh, right, these have some default arguments

Reply:
Putting them to the last minimizes the impact of API change in the users. I also prefer inputs first outputs last but perhaps we  can switch it when plugin migration is complete.